staging/traefik/templates/deployment.yaml

```diff
@@ -174,7 +174,7 @@ spec:
       {{- if and .Values.ssl.enabled (not .Values.ssl.upstream) }}
       - name: ssl
         secret:
-          secretName: {{ template "traefik.fullname" . }}-default-cert
+          secretName: traefik-certificate
```

Comment:
IMHO we should enable to define this secret-name in `values.yaml` but this one is not a blocker

Reply:
https://jira.mesosphere.com/browse/DCOS-59044